test/unit/android_test.rb

```diff
@@ -6,92 +6,91 @@ class AndroidTest < Minitest::Test
   end
 
   test "detect android cupcake (1.5)" do
-    @browser.ua = $ua['ANDROID_CUPCAKE']
-    assert @browser.android?
-    assert @browser.android?(1.5)
+    @browser.ua = Browser['ANDROID_CUPCAKE']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.